element_lab/project.py

```diff
@@ -0,0 +1,126 @@
+import importlib
+import inspect
+
+import datajoint as dj
+
+schema = dj.Schema()
+
+_linking_module = None
+
+
+def activate(
+    schema_name,
+    *,
+    create_schema=True,
+    create_tables=True,
+    linking_module=None,
+):
+    """
+    activate(schema_name, create_schema=True, create_tables=True, linking_module=None)
+        :param schema_name: schema name on the database server to activate the
+                            `lab` element
+        :param create_schema: when True (default), create schema in the
+                              database if it does not yet exist.
+        :param create_tables: when True (default), create tables in the
+                              database if they do not yet exist.
+        :param linking_module: a module (or name) containing the required
+                            dependencies to activate the `event` element:
+        Upstream tables:
+            + Lab: table defining 
+            + User: table defining user/personnel/experimenter to be associated with Project.
+            + Protocol: 
+    """
+    if isinstance(linking_module, str):
+        linking_module = importlib.import_module(linking_module)
+    assert inspect.ismodule(linking_module), (
+        "The argument 'dependency' must" + " be a module or module name"
+    )
+
+    global _linking_module
+    _linking_module = linking_module
+
+    schema.activate(
+        schema_name,
+        create_schema=create_schema,
+        create_tables=create_tables,
+        add_objects=_linking_module.__dict__,
+    )
+
+
+@schema
+class Project(dj.Manual):
+    definition = """# Top-level grouping of studies and experiments to investigate a scientific question
+    project                                  : VARCHAR(24)                 # abbreviated project name
+    ---
+    project_title                            : VARCHAR(1024)               # full project title and/or description

```

Comment:
Based on @dimitri-yatsenko suggestion of 255 instead of 256, should we changes these too?